etna/packages/etna-js/utils/cancellable_helpers.jsx

```diff
@@ -3,14 +3,10 @@ import {Cancellable, cancelledAsMaybe} from "./cancellable";
 import {useState, useEffect, useCallback} from 'react';
 
 export function useAsyncCallback(fn, deps, cleanup = () => null) {
-  const [_, setContext] = useState(() => new Cancellable());
+  const context = useContext(new Cancellable());
   const cancel = useCallback(() => {
```

Comment:
The idea of this callback, `cancel` is to execute `cancel()` on whatever current Cancellable context exists, returning a fresh one.  So essentially each time a context should be "reset" this function is called to start a fresh new context.